net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/general.xml

```diff
@@ -59,4 +59,16 @@
         <type>checkbox</type>
         <help>If set to true, do no set the "host" tag in the Telegraf agent.</help>
     </field>
+    <field>
+        <id>general.quiet</id>
+        <label>Quiet</label>
```

Comment:
Maybe change label to "Quiet Log" since it's not clear at first look that it's related to logging